src/main/java/org/folio/circulation/infrastructure/storage/ActualCostStorageRepository.java

```diff
@@ -0,0 +1,31 @@
+package org.folio.circulation.infrastructure.storage;
+
+import static org.folio.circulation.support.http.ResponseMapping.forwardOnFailure;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.circulation.domain.ActualCostRecord;
+import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.CollectionResourceClient;
+import org.folio.circulation.support.http.client.ResponseInterpreter;
+import org.folio.circulation.support.results.Result;
+
+public class ActualCostStorageRepository {
```

Comment:
The name of the entity is ActualCostRecord and we don't include the word Storage in repository names in mod-circulation - ItemRepository, InstanceRepository etc. So it should be ActualCostRecordRepository

Reply:
It is changed, thanks.